install.rst

```diff
@@ -26,11 +26,12 @@ FreeBSD and OpenBSD, or build Zeek yourself.
 Docker Images
 =============
 
-We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeek
+We provide official Docker images on Docker Hub at https://hub.docker.com/u/zeek.
 
     * For the latest feature release: ``docker pull zeek/zeek:latest``
     * For the latest LTS release: ``docker pull zeek/zeek:lts``
-    * For a specific release: ``docker pull zeek/zeek:5.0.0-rc1``
```

Comment:
We don't push `-rc1` - I sometimes wished we did though. Worth it?

Reply:
Yeah, agree.